scripts.js

```diff
@@ -13,50 +14,43 @@ addCopy.classList.add("fas", "fa-copy", "copy", "icon", "push");
 addDelete.classList.add("fas", "fa-times", "delete", "icon", "edit");
 
 // focus curson in input on load
-window.onload = getInput.focus();
+window.onload = inputElement.focus();
 
 // add return key listen and call addItem function
-getInput.addEventListener('keydown', function() {
+inputElement.addEventListener('keydown', function() {
   if (event.code == 'Enter') {
     addItem();
-    getInput.value = "";
-    getInput.focus();
+    inputElement.value = "";
+    inputElement.focus();
   }
 });
 
 // add event listeners
-function copyListener() {
-  const allCopyIcons = document.querySelectorAll('.copy');
-  if (allCopyIcons.length > 1){
-    allCopyIcons.forEach(e => e.addEventListener('click', copy));
-  } else {
-    allCopyIcons[0].addEventListener('click', copy)
-  };
-}
-
-function copy(e) {
-  const itemText = e.target.previousSibling.innerText;
+const copy = function(e) {
```

Comment:
Now it gets a bit tricky, so bear with me...

You can't use `itemText.select()` since [that only works on input elements](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/select).

Since the element that contains your text is a plain HTML element, not an Input one, this kinda screws with you since you now can't select the text you'll eventually copy to the clipboard.

Well... not screwed _screwed_, just means you'll have to go another route.

To make this work, you'll rely on the [select](https://developer.mozilla.org/en-US/docs/Web/API/Window/getSelection) and [range](https://developer.mozilla.org/en-US/docs/Web/API/Document/createRange) methods. _(basically: select a range)_

You first need to create a range, append the node you want selected _(the whole node, it's clever enough to take it's text content)_, add the range to the selection.

After the above is done, you can use your existing code to do the actual copying to the clipboard.

Here's the refactored `copy` listener method:
```js
const copy = event => {
  const range = document.createRange(); // create the range
  const selection = window.getSelection(); // create the selection

  selection.removeAllRanges(); // optional, make sure the selection so you don't get unexpected this in the clipboard
  range.selectNodeContents(event.target.previousElementSibling); // add the node to the range you want to copy
  selection.addRange(range); // add the range to selection
  document.execCommand('copy'); // copy to the clipboard. this will copy everything in your selection
};